drivers/storage/digitalocean/executor/digitalocean_executor.go

```diff
@@ -0,0 +1,120 @@
+// +build !libstorage_storage_executor libstorage_storage_executor_digitalocean
+
+package executor
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"regexp"
+
+	gofig "github.com/akutz/gofig/types"
+	"github.com/codedellemc/libstorage/api/registry"
+	"github.com/codedellemc/libstorage/api/types"
+	do "github.com/codedellemc/libstorage/drivers/storage/digitalocean"
+	doUtils "github.com/codedellemc/libstorage/drivers/storage/digitalocean/utils"
+)
+
+var (
+	diskPrefix = regexp.MustCompile(
+		fmt.Sprintf("^%s(.*)", do.VolumePrefix))
+	diskSuffix = regexp.MustCompile("part-.*$")
+)
+
+type driver struct {
+	config gofig.Config
+}
+
+func init() {
+	registry.RegisterStorageExecutor(do.Name, newDriver)
+}
+
+func newDriver() types.StorageExecutor {
+	return &driver{}
+}
+
+func (d *driver) Name() string {
+	return do.Name
+}
+
+func (d *driver) Init(ctx types.Context, config gofig.Config) error {
+	d.config = config
+	return nil
+}
+
+func (d *driver) InstanceID(
+	ctx types.Context, opts types.Store) (*types.InstanceID, error) {
+	return doUtils.InstanceID(ctx)
+}
+
+func (d *driver) NextDevice(
+	ctx types.Context, opts types.Store) (string, error) {
+	return "", types.ErrNotImplemented
+}
+
+func (d *driver) LocalDevices(
```

Comment:
By that I'm taking it that you mean that `LocalDevices` needs to filter out any mounted drives that are not actually DigitalOcean volumes? That will require https://github.com/digitalocean/godo/pull/122 being merged so that there's an easy API call to check each mounted volume